src/locale/dim.json

```diff
@@ -936,5 +938,8 @@
     "Voltron": "voltron (default)",
     "VoltronDescription": "voltron is an auto-updated collection of PvE and PvP wish list rolls from Mercules904 and pandapaxxy. We ship with this by default.",
     "WishListNotes": "Wish List Notes: {{notes}}"
-  }
+  },
+  "move-canceled": "Move canceled",
+  "no-space": "",
+  "wrong-level": ""
```

Comment:
weird side effect or?? what are these

Reply:
Yeah this is a weird side effect of making `DimError` count for i18next-scanner.